libraries/abstractions/secure_sockets/include/iot_secure_sockets.h

```diff
@@ -257,6 +257,45 @@ Socket_t SOCKETS_Socket( int32_t lDomain,
                          int32_t lProtocol );
 /* @[declare_secure_sockets_socket] */
 
+/**
+ * @brief Bind a TCP socket.
+ *
+ * See the [FreeRTOS+TCP networking tutorial]
+ * (https://freertos.org/FreeRTOS-Plus/FreeRTOS_Plus_TCP/TCP_Networking_Tutorial.html)
+ * for more information on TCP sockets.
+ *
+ * See the [Berkeley Sockets API]
+ * (https://en.wikipedia.org/wiki/Berkeley_sockets#Socket_API_functions)
+ * in wikipedia
+ *
+ * @sa SOCKETS_Bind()
+ * A pre-configured source port allows customers to bind to the specified local port instead of ephemeral port
+ * for security and packet filter reasons.
+ *
+ * Limitations:
+ *
+ *   i.  The caller of SOCKETS_Bind() API should make sure the socket address has the correct local IP address for the interface.
+ *   ii. The LWIP stack has ephemeral port range from 49152 to 65535, it is better avoid the source port in this range.
```

Comment:
Minor grammar fix
```suggestion
 *   ii. The LWIP stack has ephemeral port range from 49152 to 65535, it is better to avoid the source port in this range.
```

Reply:
done